### ISSUE TITLE
Cannot find module @babel/register

### ISSUE THREAD

**Issue by samtstern** [brianneisler/firemin#4]
I was trying to use `firemin` so I did:

```
npm install -g firemin
```

And then:
```
firemin minimize rules/solution-rbac/step5.rules
```

I get:
```
Error: Cannot find module '@babel/register'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at Function.Module._preloadModules (module.js:754:12)
    at preloadModules (bootstrap_node.js:488:38)
    at startup (bootstrap_node.js:201:9)
    at bootstrap_node.js:625:3
```

I tried to clone this repo and then `npm` link a local install into my globals but I get a different errror:
```
$ firemin minimize rules/solution-rbac/step5.rules
/usr/local/google/home/samstern/Scratch/firemin/node_modules/@babel/core/lib/config/files/plugins.js:152
    throw e;
    ^

Error: Cannot find module '@babel/plugin-proposal-object-rest-spread' from '/usr/local/google/home/samstern/Projects/firebase/snippets-rules'
    at Function.module.exports [as sync] (/usr/local/google/home/samstern/Scratch/firemin/node_modules/resolve/lib/sync.js:74:15)
    at resolveStandardizedName (/usr/local/google/home/samstern/Scratch/firemin/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
    at resolvePlugin (/usr/local/google/home/samstern/Scratch/firemin/node_modules/@babel/core/lib/config/files/plugins.js:54:10)
    at loadPlugin (/usr/local/google/home/samstern/Scratch/firemin/node_modules/@babel/core/lib/config/files/plugins.js:62:20)
    at createDescriptor (/usr/local/google/home/samstern/Scratch/firemin/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at items.map (/usr/local/google/home/samstern/Scratch/firemin/node_modules/@babel/core/lib/config/config-descriptors.js:109:50)
    at Array.map (<anonymous>)
    at createDescriptors (/usr/local/google/home/samstern/Scratch/firemin/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPluginDescriptors (/usr/local/google/home/samstern/Scratch/firemin/node_modules/@babel/core/lib/config/config-descriptors.js:105:10)
    at alias (/usr/local/google/home/samstern/Scratch/firemin/node_modules/@babel/core/lib/config/config-descriptors.js:63:49)
```

I know zero about babel but overall I think there's some issue with how `firemin` is being packaged for release.  Would be happy to send a PR to fix this I just am not sure where to begin at the moment.

**Comment by samtstern** [comment]
Update: I was able to use `/bin/firemin` from a local build of this repository just fine ... shrunk my sample rules from 1855B to 1125B!

**Comment by brianneisler** [comment]
@samtstern thanks for reporting this issue! I'm using firemin as a local dependency in my own project and hadn't tried it globally which is likely why i missed this. I agree that this is likely a simple packaging issue. Will take a look at it and release a new version today.

**Comment by brianneisler** [comment]
@samtstern this issue has been fixed in release v0.0.4 https://github.com/brianneisler/firemin/releases/tag/v0.0.4

**Comment by samtstern** [comment]
@brianneisler just tested out the new version, works great!
